Sources/NestKit/GitHub/GitHubURLBuilder.swift

```diff
@@ -21,10 +21,12 @@ public enum GitHubURLBuilder {
             throw InvalidURLError(url: url)
         }
 
-        switch version {
-        case .latestRelease:
+        switch (version, hasExcludedVersion) {
+        case (.latestRelease, true):
+            return baseURL.appending(components: "repos", owner, repository, "releases")
```

Comment:
Since the latest version is returned in descending order, I think pagination support is not necessary for now.

Reply:
Good information!
So, it would be kinder if the information is written as inline comment!

Reply:
Thanks!
I've made the corrections here: https://github.com/mtj0928/nest/pull/33/commits/3d93849157361f32c97fa9c60dd04d68c1ea84c1#diff-969fea1b46cf42d9b3481f2f3953a9892cf8ec505f467d1464ed9a49ee64c42aR27